src/__test__/dependencies.test.ts

```diff
@@ -190,4 +198,65 @@ describe('dependencies.js tests', () => {
     );
     expect(bump10.type).toBe(BumpType.PRERELEASE);
   });
+
+  it('Should return a valid bump recommendation for a monorepo packages', async () => {
```

Comment:
nit: typo in the test name 😅 
`for a monorepo packages` ➡️  `for monorepo packages`